### PR TITLE
[0.9.x] Fix examples configuration and instructions

### PR DESCRIPTION
**Pull request by jstastny-cz**
Cherry-pick of #175

### CODE REVIEW

File: process-usertasks-with-security-quarkus/README.md [hunk 1]
```diff
@@ -86,37 +99,37 @@ To make use of this application it is as simple as putting a sending request to
...
 
 where `{uuid}` is the id of the given approval instance
 
 
 ### Complete first line approval task
 
-```
-curl -X POST -d '{"approved" : true}' -H 'Content-Type:application/json' -H 'Accept:application/json' http://localhost:8080/approvals/{uuid}/firstLineApproval/{tuuid}?user=admin&group=managers'
+```sh
+curl -u john:john -X POST -d '{"approved" : true}' -H 'Content-Type:application/json' -H 'Accept:application/json' http://localhost:8080/approvals/{uuid}/firstLineApproval/{tuuid}?user=john&group=managers'
```
Comment (radtriste) [inline process-usertasks-with-security-quarkus/README.md]: you are missing an extra `'` before the url or ?

File: process-usertasks-with-security-springboot/README.md [hunk 2]
```diff
@@ -76,37 +76,37 @@ To make use of this application it is as simple as putting a sending request to
...
 
 where `{uuid}` is the id of the given approval instance
 
 
 ### Complete first line approval task
 
-```
-curl -X POST -d '{"approved" : true}' -H 'Content-Type:application/json' -H 'Accept:application/json' http://localhost:8080/approvals/{uuid}/firstLineApproval/{tuuid}?user=admin&group=managers'
+```sh
+curl -u john:john -X POST -d '{"approved" : true}' -H 'Content-Type:application/json' -H 'Accept:application/json' http://localhost:8080/approvals/{uuid}/firstLineApproval/{tuuid}?user=john&group=managers'
```
Comment (radtriste) [inline process-usertasks-with-security-springboot/README.md]: same here about the missing `'`

**Review by radtriste — APPROVED** [review]
2 small things. otherwise lgtm

File: process-usertasks-with-security-quarkus/README.md (hunk 1 above)
Comment (jstastny-cz) [reply process-usertasks-with-security-quarkus/README.md]:
```suggestion
curl -u john:john -X POST -d '{"approved" : true}' -H 'Content-Type:application/json' -H 'Accept:application/json' 'http://localhost:8080/approvals/{uuid}/firstLineApproval/{tuuid}?user=john&group=managers'
```

File: process-usertasks-with-security-springboot/README.md (hunk 2 above)
Comment (jstastny-cz) [reply process-usertasks-with-security-springboot/README.md]:
```suggestion
curl -u john:john -X POST -d '{"approved" : true}' -H 'Content-Type:application/json' -H 'Accept:application/json' 'http://localhost:8080/approvals/{uuid}/firstLineApproval/{tuuid}?user=john&group=managers'
```

**Pull request merged**